schema/crdb/blueprint-sled-last-used-ip/up02.sql

```diff
@@ -0,0 +1,59 @@
+-- Working with INET values in SQL is "fun".
```

Comment:
This migration query is quite horrifying IMO. I would love suggestions to make it less so.